tests/integration/test_data/npm_packages.yaml

```diff
@@ -0,0 +1,191 @@
+# Test data for npm packages
+#
+# npm package without dependencies
+# repo: The URL for the upstream git repository
+# ref: A git reference at the given git repository
+# expected_files: Expected source files <relative_path>: <file_URL>
+# expected_deps_files: Expected dependencies files (empty)
+# response_expectations: Parts of the Cachito response to check
+# content_manifest: PURLs for image contents part
+without_deps:
+  repo: https://github.com/cachito-testing/cachito-npm-without-deps.git
+  ref: 2f0ce1d7b1f8b35572d919428b965285a69583f6
+  pkg_managers: ["npm"]
+  expected_files:
+    app: https://github.com/cachito-testing/cachito-npm-without-deps/tarball/2f0ce1d7b1f8b35572d919428b965285a69583f6
+    deps/npm: null
+  response_expectations:
+    dependencies: []
+    packages:
+      - dependencies: []
+        name: "cachito-npm-without-deps"
+        type: "npm"
+        version: "1.0.0"
+  content_manifest:
+  - purl: "pkg:github/cachito-testing/cachito-npm-without-deps@2f0ce1d7b1f8b35572d919428b965285a69583f6"
+# npm package with dependencies package.json
+# repo: The URL for the upstream git repository
+# ref: A git reference at the given git repository
+# expected_files: Expected source files <relative_path>: <file_URL>
+# expected_deps_files: Expected dependencies files <relative_path>
+# response_expectations: Parts of the Cachito response to check
+# content_manifest: PURLs for image contents part
+with_deps:
+  repo: https://github.com/cachito-testing/cachito-npm-with-deps.git
+  ref: dc4c274b40f46b7e50f3327facf6929424282b0f
+  pkg_managers: ["npm"]
+  response_expectations:
+    packages:
+      - dependencies:
+          - dev: false
+            name: "type-detect"
+            replaces: null
+            type: "npm"
+            version: "4.0.8"
+          - dev: false
+            name: "test-npm-dep"
+            replaces: null
+            type: "npm"
+            version: "git+https://elinah@bitbucket.org/elinah/test-npm-dep.git#b952715c0706f30d92a7253dcdadc510a618eca5"
+          - dev: false
+            name: "pathval"
+            replaces: null
+            type: "npm"
+            version: "1.1.1"
+          - dev: true
+            name: "npm-test-project"
+            replaces: null
+            type: "npm"
+            version: "git+https://gitlab.com/eakhmano-test-group/eakhmano-test-subgroup/subsubgroup/npm-test-project.git#88114c48d7d894f7e2306adf06137b096e9cdf29"
+          - dev: false
+            name: "get-func-name"
+            replaces: null
+            type: "npm"
+            version: "2.0.0"
+          - dev: false
+            name: "fecha"
+            replaces: null
+            type: "npm"
+            version: "git+https://github.com/taylorhakes/fecha.git#e5f18494528622f7f3c37434b10c6b8d03c9597b"
```

Comment:
For the yarn test, this one is a plain https url. I didn't find one for npm, should we change this one to be plain https?

Reply:
Good catch, I thought that Plain URL will be used by test-npm-dep: https://github.com/cachito-testing/cachito-npm-with-deps/blob/master/package.json
I'll change fecha dep in the same way as it is in yarn test.